joomla/joomla-cms
# [com_fields] Fixing ACL inheritance for fields and fieldgroups

Pull Request for Issue #13662.

### Summary of Changes
When we added context support for fieldgroups, the assetname wasn't taken care of properly and thus ACL inheritance broke.

### Testing Instructions
* Create some fieldgroups and assign some fields to it.
* Play with ACL settings. Settings should inherit from the component to fieldsgroups to fields. That means for example if you allow or deny the "Edit Custom Field Value" action in a field group, the fields in it should take that permission.

### Documentation Changes Required
None
